Source/OpenTK/Input/JoystickState.cs

```diff
@@ -41,13 +41,13 @@ public struct JoystickState : IEquatable<JoystickState>
         // If we ever add more values to JoystickAxis or JoystickButton
         // then we'll need to increase these limits.
         internal const int MaxAxes = (int)JoystickAxis.Last + 1;
-        internal const int MaxButtons = (int)JoystickButton.Last + 1;
+        internal const int MaxButtons = 128;
```

Comment:
With the code you are doing bellow, the MaxButton count should be 64, ie. `(Int64)1 << index` will get out of bounds and loop back around after 63